libs/designsystem/src/lib/components/button/button.component.ts

```diff
@@ -74,7 +74,7 @@ export class ButtonComponent implements AfterContentInit {
   @Input() expand: 'full' | 'block';
   @Input() isFloating: boolean = false;
   @Input()
-  size: ButtonSize = ButtonSize.MD;
+  size: ButtonSize | 'sm' | 'md' | 'lg' = ButtonSize.MD;
```

Comment:
```suggestion
  size: ButtonSize | `${ButtonSize}` = ButtonSize.MD;
```

Can this also be changed like the rest?